llvm/lib/Target/AMDGPU/AMDGPUGISel.td

```diff
@@ -297,7 +297,7 @@ def : GINodeEquiv<G_AMDGPU_S_BUFFER_LOAD_UBYTE, SIsbuffer_load_ubyte>;
 def : GINodeEquiv<G_AMDGPU_S_BUFFER_LOAD_SSHORT, SIsbuffer_load_short>;
 def : GINodeEquiv<G_AMDGPU_S_BUFFER_LOAD_USHORT, SIsbuffer_load_ushort>;
 
-def : GINodeEquiv<G_FPTRUNC_ROUND, SIfptrunc_round>;
+def : GINodeEquiv<G_INTRINSIC_FPTRUNC_ROUND, SIfptrunc_round>;
```

Comment:
SIfptrunc_round shouldn't be necessary anymore, all code using it should be deleted and use the generic version

Reply:
I am ot clear what is " the generic version" here to replace SIfptrunc_round.

Reply:
Apparently it's not defined. I don't know why we have SIfptrunc_round, it should just be defined in TargetSelectionDAG.td without the target prefix

Reply:
Initial definition of the SDNode is for AMDGPUISD::FPTRUNC_ROUND, and thus in AMDGPU backend. Now that we are using ISD::FPTRUNC_ROUND, we should move the related definitions into TargetSelectionDAG.td.
Thanks for pointing out.